Shared/ViewModels/AdminDashboard/ServerUserAdminViewModel.swift

```diff
@@ -16,49 +16,43 @@ final class ServerUserAdminViewModel: ViewModel, Eventful, Stateful, Identifiabl
     // MARK: Event
 
     enum Event {
-        case success
-    }
-
-    // MARK: BackgroundState
-
-    enum BackgroundState {
-        case updating
```

Comment:
We should still use `BackgroundState.updating`, as this indicates that the policy update is being made. It shouldn't be a primary `State` as we don't have any larger changes in design during that state and shouldn't be mixed up with `State.content`.